src/reducers/__tests__/query.spec.jsx

```diff
@@ -372,6 +373,7 @@ describe( 'reducer:query', () => {
         from: 0,
         page: 1,
         searchAfter: '',
+        dataNormalization: undefined,
```

Comment:
I dont think this test needs to be updated.

this line [here](https://github.com/cfpb/ccdb5-ui/blob/main/src/utils/reducers.jsx#L49) should be changed to 
      defaultVal: types.MODE_TRENDS

so that if we accidentally pass in an invalid tab name (coming from url or something, it makes the default to be the Trends tab.

Reply:
Updated in https://github.com/cfpb/ccdb5-ui/pull/386/commits/1dc07cbb61cc6f8c1ddd316ff792c31d0f122911

Reply:
The test is updated to that of the Trends signature. That makes sense, right?

Reply:
Yes it does. its that we want  it to be on the trends tab as defaultl.